tower-grpc-build/src/lib.rs

```diff
@@ -119,6 +119,29 @@ impl prost_build::ServiceGenerator for ServiceGenerator {
     }
 }
 
+/// Extension trait for importing generated types into `codegen::Scope`s.
+trait ImportType {
```

Comment:
I'm not sure I follow why a trait is needed instead of inherent fns on `Scope` and `Module`.

Reply:
@carllerche we can't define inherent impls for `Scope` and `Module` here, since they come from the `codegen` crate; and I didn't want to add any of this to `codegen`, since it's _very_ `tower-grpc`-specific. 

It probably could have just been bare functions like  `fn import_type(module: &mut Module, ty: &str, level: usize)`, but then they'd have to have different names since they can't be overloaded for `Module` and `Scope`, and I thought having the `Module`/`Scope` in the method-receiver position communicated the intent more clearly --- it's the module/scope that imports the type...

Reply:
Ah yes, I forgot it was in a different crate.